packages/block-library/src/preformatted/editor.scss

```diff
@@ -0,0 +1,4 @@
+.wp-block-preformatted {
+	overflow-x: auto;
```

Comment:
The `style.scss` file always gets loaded into the editor, so if a rule exists in that file, it does not need to be replicated in `editor.scss`.

Reply:
Oh, yes, true that. I'll fix accordingly.